src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -687,14 +702,26 @@ define([
                         this.renderPreview(data);
                     }.bind(this));
 
+                    this.sandbox.on('sulu.preview.changes', this.handleChanges.bind(this));
+
                     this.sandbox.on('sulu.preview.initialize', function(data, restart) {
+                        if (this.previewMode === 'auto') {
+                            // if mode is auto init immediately
+                            this.sandbox.emit('sulu.preview.initialize.force', data, restart);
```

Comment:
It just feels very strange to me, you have to emit both events for the preview to be initialized. Isn't there a better naming? Maybe call the outer event `sulu.preview.render` instead, since the preview is not really initialized?